tests/h/schemas/admin_group_test.py

```diff
@@ -50,11 +58,69 @@ def test_it_raises_if_required_field_missing(self, group_data, bound_schema, req
         with pytest.raises(colander.Invalid, match='.*{field}.*'.format(field=required_field)):
             bound_schema.deserialize(group_data)
 
-    def test_it_allows_when_optional_field_missing(self, group_data, bound_schema):
-        group_data.pop('description')
+    @pytest.mark.parametrize('optional_field', (
+        'description',
+        'origins'
+    ))
+    def test_it_allows_when_optional_field_missing(self, group_data, bound_schema, optional_field):
+        group_data.pop(optional_field)
 
         bound_schema.deserialize(group_data)
 
+    @pytest.mark.parametrize('input_origins,expected_origins', (
+        ('http://www.foo.com', ['http://www.foo.com']),
+        ('http://www.foo.com\r\nhttps://www.foo.com', ['http://www.foo.com', 'https://www.foo.com']),
+        ('http://www.foo.com   ', ['http://www.foo.com']),
+        ('http://www.foo.com\nhttps://www.foo.com', ['http://www.foo.com', 'https://www.foo.com']),
+    ))
+    def test_it_splits_origins_by_line(self, group_data, bound_schema, input_origins, expected_origins):
```

Comment:
Are we validating that the origin is actually an absolute URL anywhere? If not, I think we should add that. I would suggest that we might also want to make it clear somehow that only the origin part is respected at present.